src/main/osd/osd.c

```diff
@@ -545,6 +548,19 @@ void osdInit(displayPort_t *osdDisplayPortToUse, osdDisplayPortDevice_e displayP
     }
 }
 
+#ifdef USE_GPS_LAP_TIMER
+void printLapTime(char *buffer, uint32_t timeMs) {
```

Comment:
Mark `timeMs` as `const` to show to others that you want it to be immutable inside of the function body. Otherwise you could just use `timeMs` to increment the time by 5, as it's already a copy of the original input.
```suggestion
void printLapTime(char *buffer, const uint32_t timeMs) {
```

Reply:
Sorry, I didn't want to annoy you with this detail. Maybe I did not fully grasp your intentions. I don't want this to seem like you absolutely need to do what I suggest. If you have strong feelings against this, just change it back to the original way you wanted it to be. I'll just approve, as the PR is super solid (always has been).